### PR TITLE
Updating Undocumented

### PR DESCRIPTION
**Pull request by DrinkWater623**
Mainly under the Entity Components

- Updated descriptions
- Fixed typos
- Corrected data types
- Added 'default' where needed
- Added missing parameters
- Removed invalid parameters
- Added and/or updated examples

Used information from MS-Learn component pages.  
* *Except for entity_placer.json, added logical title and description to the items-array info for the dispense_on/use_on*

### CODE REVIEW

**Review by DaanV2 — CHANGES_REQUESTED** [review]

File: source/behavior/entities/format/components/is_stackable.json [hunk 1]
```diff
@@ -4,11 +4,5 @@
   "type": "object",
   "title": "Is Stackable",
   "additionalProperties": false,
-  "properties": {
-    "value": {
-      "title": "Value",
-      "description": "UNDOCUMENTED.",
-      "type": "boolean"
-    }
-  }
```
[the remark this answers is not in the document]
Comment (DrinkWater623) [reply source/behavior/entities/format/components/is_stackable.json:13]: So do I just remove it, or do you?   I am kinda new to this github thing.

File: source/behavior/entities/format/components/shooter.json [hunk 2]
```diff
@@ -17,12 +18,27 @@
...
+      "type": "array",
+      "items": { "type": "string", "description": "Projectiles that can be used by the shooter", "title": "Projectile"}
+    },
+    "sound": {
+      "title": "Sound",
       "type": "string",
-      "description": "UNDOCUMENTED.",
-      "$comment": "UNDOCUMENTED",
-      "examples": ["dragonfireball"]
+      "description": "Sound that is played when the shooter shoots a projectile."
```
[the remark this answers is not in the document]
Comment (DrinkWater623) [reply source/behavior/entities/format/components/shooter.json:41]: I got everything from [Microsoft](https://learn.microsoft.com/en-us/minecraft/creator/reference/content/entityreference/examples/entitycomponents/minecraftcomponent_shooter).  Is this not correct, it is there.  Where are you looking?

File: source/behavior/entities/format/components/is_stackable.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (DaanV2) [reply source/behavior/entities/format/components/is_stackable.json:13]: You removed in your code 🙂, So for this case you need to re-add it to the file 😄

File: source/behavior/entities/format/components/shooter.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (DaanV2) [reply source/behavior/entities/format/components/shooter.json:41]: Sorry right, was looking at the minecraft:shooter for items

**Comment by DrinkWater623** [comment]
I deleted all changes for that file.  Well I hope I did it right.

File: source/behavior/entities/format/components/is_stackable.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (DrinkWater623) [reply source/behavior/entities/format/components/is_stackable.json:13]: Okay, I think this is where the reply goes...  I deleted my changes for is_stackable.

**Comment by DrinkWater623** [comment]
Getting confused about where comments go... anyways, deleted the is_stackable file from the PR.  I think.

File: source/behavior/entities/format/components/is_stackable.json [hunk 3]
```diff
@@ -4,11 +4,5 @@
...
+  "properties": {}
```
Comment (DaanV2) [inline source/behavior/entities/format/components/is_stackable.json:7]:
```suggestion
"properties": {
    "value": {
      "title": "Value",
      "description": "UNDOCUMENTED.",
      "type": "boolean"
    }
  }
```

File: source/behavior/entities/format/components/is_stackable.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "type": "object",
   "title": "Is Stackable",
   "additionalProperties": false,
-  "properties": {
+"properties": {
```
Comment (DaanV2) [inline source/behavior/entities/format/components/is_stackable.json:7]:
```suggestion
  "properties": {
```

**Review by DaanV2 — APPROVED** [review]

**Pull request merged**